.github/workflows/main.yml

```
@@ -24,7 +24,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [20.x]
```

test-node must execute on all versions >= 18.x
```suggestion
        node-version: [18.x, 20.x]
```